client/call.go

```diff
@@ -39,3 +40,133 @@ func (c *Call) resetClientCmdID(clock Clock) {
 		Random:   rand.Int63(),
 	}
 }
+
+// Method returns the name of the database command for the call.
+func (c *Call) Method() string {
+	return c.Args.Method()
+}
+
+// GetCall returns a Call object initialized to get the value at key.
+func GetCall(key proto.Key, reply *proto.GetResponse) *Call {
```

Comment:
Not a fan of this. How much extra work is the transition without it? Problem is this API really makes it look like the reply is expected. There's an element of surprise here that bothers me. I'd drop it.